tests/unit/specs/components/HelpSection.spec.js

```diff
@@ -1,73 +1,59 @@
-import { mount, createLocalVue, config } from '@vue/test-utils'
+import { mount, createLocalVue } from '@vue/test-utils'
 import HelpSection from '@/components/HelpSection.vue'
-import VueI18n from 'vue-i18n'
-import Buefy from 'buefy'
+import VueVocabulary from '@creativecommons/vue-vocabulary/vue-vocabulary.common'
 
 describe('HelpSection.vue', () => {
     let wrapper
 
-    // Always creates a shallow instance of component
     beforeEach(() => {
         const localVue = createLocalVue()
-        localVue.use(VueI18n)
-        localVue.use(Buefy)
-        const messages = require('@/locales/en.json')
-        const i18n = new VueI18n({
-            locale: 'en',
-            fallbackLocale: 'en',
-            messages: messages,
-            silentTranslationWarn: true
-        })
-
-        config.mocks.i18n = i18n
-
-        config.mocks.i18n.$t = (key) => {
-            return key
-        }
+        localVue.use(VueVocabulary)
         wrapper = mount(HelpSection, {
             localVue,
-            i18n
-        })
-        wrapper.setData({
-            sixLicensesImg: '<img src="../assets/license-openness-scale.png">',
-            modals: {
-                1: {
-                    status: false,
-                    title: 'what-are-cc-licenses'
-                },
-                2: {
-                    status: false,
-                    title: 'how-licenses-work'
-                },
-                3: {
-                    status: false,
-                    title: 'what-icons-mean'
-                },
-                4: {
-                    status: false,
-                    title: 'considerations-before-licensing'
-                },
-                5: {
-                    status: false,
-                    title: 'how-formally-license'
-                },
-                6: {
-                    status: false,
-                    title: 'six-cc-licenses'
-                },
-                7: {
-                    status: false,
-                    title: 'how-licenses-communicated'
-                },
-                8: {
-                    status: false,
-                    title: 'what-free-culture-license'
-                },
-                9: {
-                    status: false,
-                    title: 'look-earlier-license-ver'
+            mocks: {
+                $t: key => key
+            },
+            data: () => ({
+                sixLicensesImg: '',
+                modals: {
```

Comment:
The reason I didn't want to use an `Array`, I think, was I wanted the keys to start at 1, not at 0. With an array, I can't check if the modal is open with `if modals[openModal]`, because when the open modal is at index 0, this returns false. I've solved it by checking `if openModal !== null` instead, and now it looks much cleaner, I think.

Reply:
You can always create an `Array` with n+1 elements and leave the index 0 empty.